sensor_msgs/Image image
---
sensor_msgs/Image predict_image
